docs/content/install/using-docker.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using the Athens Docker images
+description: Information about Athens' Docker images
+weight: 1
+---
+
+Whether setting Athens up using [Kubernetes](./install-on-kubernetes) or using the [Walkthrough](/Walkthrough), you'll most likely be using one of the images that the Athens project produces. This document details what images are available, and has a recap from the Walkthrough of how to use them on their own.
+
+---
+
+## Available Docker images
+
+The Athens project produces two docker images, available via [Docker Hub](https://hub.docker.com/) 
+
+1. A release version as [`gomods/athens`](https://hub.docker.com/gomods/athens), each tag corresponds with an Athens [release](https://github.com/gomods/athens/releases), e.g. `v0.7.1`
+2. A tip version, as [`gomods/athens-dev`](https://hub.docker.com/r/gomods/athens-dev), tagged with every commit to `master`, e.g. `1573339`
```

Comment:
I think it's worth adding point `3` here (or addition to the first point since it's the same docker hub repo) which is 

- a `canary` version that is frequently mutated to be the latest commit from master.  To install run `docker pull gomods/athens:canary`

Reply:
Is there some sort of schedule for `canary` releases?

Reply:
@twexler on every merge to master

Reply:
Done.